spec/game_spec.rb

```diff
@@ -0,0 +1,39 @@
+require_relative '../game'
+
+describe 'Unit testing for Game Class' do
+  before :each do
+    @game = Game.new('Y', '1994/08/01', '2010/02/05')
+  end
```

Comment:
All public methods in ALL classes should have unit tests.

- Kindly test the `can_be_archived?` method in the `Item` and `Game` classes as well.